src/Command/PelagosClearErrorAndFileCommand.php

```diff
@@ -102,4 +102,22 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
         return 0;
     }
+
+    /**
+     * Method to determine where a file is stored.
+     *
+     * @param Datafile $file
+     * @param string   $uploadDir injected
+     *
+     * @return string
+     */
+    protected function getStorageLocation($file)
+    {
+        $uploadDir = preg_quote($this->homedirPrefix . '/upload/', '/');
+        if (preg_match("/$uploadDir/", $file->getPhysicalFilePath())) {
+            return 'uploaddir';
```

Comment:
I don't like using words, maybe if they were constants, it's probably easier to return a boolean, and that can also be done with the `datastore->has` method. Which may be a bit more expensive, but this doesn't run that often.